publish.js

```diff
@@ -0,0 +1,15 @@
+var ghpages = require('gh-pages');
+var path = require('path');
+
+ghpages.publish('public',{
+	user: {
+		name: 'OpenFisca-Bot',
+	},
+	branch: 'gh-pages',
+	message: 'Auto-commit from master branch'
+	},function(err) {
+		if(err){
+			console.error(err)
+    		process.exit(1)
```

Comment:
I'm not 100% confident of the rules of indentation for a function's argument. What would a good source be for this kind of information ?